src/compiler/commandLineParser.ts

```diff
@@ -7,6 +7,39 @@
 namespace ts {
     /* @internal */
     export const compileOnSaveCommandLineOption: CommandLineOption = { name: "compileOnSave", type: "boolean" };
+
+    /* @internal */
+    export const libMap = createMapFromTemplate({

```

Comment:
Alternatively, we leave `libMap` as is, and add: `export const libs = arrayFrom(libMap.keys();`